src/main/java/frc/robot/RobotContainer.java

```diff
@@ -185,6 +186,11 @@ private void configureButtonBindings() {
         .onTrue(
             Commands.runOnce(drivetrainSubsystem::reseedSteerMotorOffsets, drivetrainSubsystem));
 
+    controllertwo
```

Comment:
We will need a way to avoid conflicts while testing things, but still support the final gameday robots.
I suggest one method for `configureGameButtonBindings()` for the driver interface you design for drivers, and a one or more different ones for test mode `configureAutoBalanceBindings()`, `configureLimelightTestBindings()` and so on.
Then each branch can make the buttons needed to test a subsystem or command and not mess up the game bindings.

Reply:
we'll work on that!